packages/node-resolve/src/index.js

```diff
@@ -190,7 +191,7 @@ export function nodeResolve(opts = {}) {
       allowExportsFolderMapping: options.allowExportsFolderMapping
     });
 
-    const importeeIsBuiltin = isBuiltinModule(importee);
+    const importeeIsBuiltin = builtinModules.includes(importee.replace('node:', ''));
```

Comment:
```suggestion
    const importeeIsBuiltin = builtinModules.includes(importee.replace(/^node:/, ''));
```
Is a bit more precise